cg/cli/add.py

```diff
@@ -226,10 +226,10 @@ def case(
         LOG.error(f"{customer_id}: customer not found")
         raise click.Abort
 
-    for panel_id in panels:
-        panel: Panel = status_db.panel(abbrev=panel_id)
+    for panel_abbreviation in panels:
```

Comment:
```suggestion
    for panel_abbreviation in panels_abbreviations:
```

Reply:
Oh, apparently we can rename the internal name for the click variable without changing the cli. I missed that

```python
@click.option("-p", "--panel", "panels", multiple=True, help="default gene panels")
```